go.mod

```diff
@@ -2,10 +2,19 @@ module github.com/vmware-tanzu/antrea
 
 go 1.12
 
+replace github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20191122085527-2c79e3ce4182
```

Comment:
I see normally `replace` is below `require`, and we already have one replace there, can move it there to be in same block

Reply:
Done.